datastore/gcloud/rest/datastore/datastore.py

```diff
@@ -0,0 +1,412 @@
+import logging
+import os
+from typing import Any  # pylint: disable=unused-import
+from typing import Dict  # pylint: disable=unused-import
+from typing import List  # pylint: disable=unused-import
+from typing import Optional  # pylint: disable=unused-import
+from typing import Union  # pylint: disable=unused-import
+
+import requests
+
+from gcloud.rest.auth import Token  # pylint: disable=no-name-in-module
+from gcloud.rest.datastore.constants import Consistency
+from gcloud.rest.datastore.constants import Mode
+from gcloud.rest.datastore.constants import Operation
+from gcloud.rest.datastore.datastore_operation import DatastoreOperation
+from gcloud.rest.datastore.entity import EntityResult
+from gcloud.rest.datastore.key import Key
+from gcloud.rest.datastore.query import BaseQuery  # pylint: disable=unused-import
+from gcloud.rest.datastore.query import QueryResultBatch
+from gcloud.rest.datastore.value import Value
+
+try:
+    import ujson as json
+except ImportError:
+    import json  # type: ignore
+try:
```

Comment:
nit: keep the linebreaks here, please -- I'm actually surprised our linting doesn't catch this, there should be two empty lines after all the imports.